menu.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <section class="headline">
+   <div class="works">
+      <h1>My Recent Works</h1>
+      <img src="images/icons/Indicator.svg" alt="Indicator"><br>
+    <div class="work-list">
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <div class="name">Multi-Post Stories Gain+Glory</div>
+          <div id="2">
+            <ul>
+              <li>Ruby On Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="content-three"> See project</div>
+      </div> 
+        </div>
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <div class="name">Multi-Post Stories Gain+Glory</div>
+          <div id="2">
+            <ul>
+              <li>RubyOnRails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="content-three"> See project</div>
```

Comment:
- [ ] According to the design, This should be a link `a` or `button` tag, and it should follow the button styling: `Active, Hover and Pressed`  I'll suggest you update this part of your code according. :+1: :smile: 
- [ ] As part of the requirement for this milestone is to try and replicate the given design as much as you can, looking at your current design, I think you need to adjust the font size of your `See Project button` to match the given design. 

- YOUR DESIGN:
![Screenshot from 2023-05-06 16-16-45](https://user-images.githubusercontent.com/107632780/236633328-137dfbec-6cfc-4e10-9985-58fd4ab82742.png)

- FIGMA DESIGN:
![Screenshot from 2023-05-06 16-19-14](https://user-images.githubusercontent.com/107632780/236633339-6480a8dc-a831-4fdd-8288-584755382eeb.png)

NB: I'll suggest you apply this to all other buttons in this section too. :+1: :wave: